lib/mosaic-routing/src/test/java/org/eclipse/mosaic/lib/routing/graphhopper/GraphHopperWeightingTest.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (c) 2020 Fraunhofer FOKUS and others. All rights reserved.
```

Comment:
Do we have to adjust this year, as this file is marked as "new"?

Reply:
No we don't **have** to do this.